### DISCUSSION TITLE
How to configure renovate to create pr only when all transitive dependencies are available in the artifactory?

### DISCUSSION THREAD

**Discussion by jakmal521** [renovatebot/renovate#23064] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Hi! I want to know whether it is possible to create a pull request only when all transitive dependencies are in the artifactory.
For some reason, in my use case, all transitive are not always available, so I would like to have pr open only when all are present.

### What you tried so far.

-

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
That's not possible, but if you set `minimumReleaseAge` to whatever days you need then you will have a higher chance of the dependencies being available.
